### DISCUSSION TITLE
fail to convert 16bit RGB images

### DISCUSSION THREAD

**Discussion by scpsc** [ImageMagick/ImageMagick#6162] [Help] [answered]
### ImageMagick version

6.9.10-23

### Operating system, version and so on

ubuntu 20.04

### Description

Hello,
I'm trying to crop a set of 16bit RGB images using the following command:
`convert -compress None "T*.tif[1001x1001+10+10]" -set filename:f "%t_1001x1001px.%e" +adjoin "%[filename:f]"`
I obtain the right size, but the images have only two bands and are shrinked in the x dimension in a strange way. When I try with an 8bit version of my images, it works, so I suspect that something is missing in my command, regarding the 16bit format.

I tested with a simpler command, involving only one file, leading to the same result:
convert T11SNB.tif -crop 1001x1001+10+10 test.tif

When using 1-band 16bit images, there is no problem, so only the combination of 16bits and RGB is problematic in this case.

I would appreciate some help!

Here is a sample of the images I'd like to convert:
[T11SNB.tif.zip](https://github.com/ImageMagick/ImageMagick/files/10969730/T11SNB.tif.zip)

![Screenshot from 2023-03-14 15-24-41](https://user-images.githubusercontent.com/4867738/225032279-b440c156-e8f5-40fd-a3db-4e76176ba72c.png)

**Comment by snibgo** [comment, marked as answer]
Your file T11SNB.tif isn't RGB. It is grayscale, with two extra samples, both "unknown" according to exiftool.

IM v7.1.0-62 reads these extra samples as alpha and meta[0]. The values in all three channels are low, all less than 0.1 on a scale of 0 to 1.

This works fine:
```
%IMG71062%magick T11SNB.tif[1001x1001+10+10] -alpha off -auto-level x.png
```
IM has only recently had the ability to read meta channels. I don't know if this has been backported to v6. I suggest you upgrade to the current v7.

**Reply by scpsc** [reply]
You're right: altough I intended to create RGB images, they are not. And converting it to RGB solves my issue.
Thanks a lot for your time and help!
